packages/broker/src/ConfigWizard.ts

```diff
@@ -1,38 +1,46 @@
 import inquirer from 'inquirer'
 import { Wallet } from 'ethers'
-import { writeFileSync, mkdirSync, existsSync } from 'fs'
+import { Config } from './config'
 import path from 'path'
+import { writeFileSync, existsSync, mkdirSync } from 'fs'
+import * as os from 'os'
+import chalk from "chalk"
 
-import { Config } from './config'
+function logger(...args: any[]) {
+    console.log(chalk.bgWhite.black(':'), ...args)
+}
 
-import * as os from 'os'
+logger.info = (...args: any[]) => {
```

Comment:
Maybe all logger methods could be functions of a logger object? If there is no need that this "plain" logger is an exceptional case.

Reply:
something like:
```typescript
function logger(...args: any[]) {
    console.log(chalk.bgWhite.black(':'), ...args)
}

logger.prototype = {
    info: (...args: any[]) => {
        console.log(chalk.bgWhite.black(':', ...args))
    },
    alert: (...args: any[]) => {
        console.log(chalk.bgYellow.black('!', ...args))
    },
    warn: (...args: any[]) => {
        console.log(chalk.bgYellow.black('!'), ...args)
    },
    error: (...args: any[]) => {
        console.log(chalk.bgRed.black('!'), ...args)
    }
}
```

Reply:
Maybe renaming the previous approach is easier?

```typescript
const logger = {
    print: (...args: any[]) => {
        console.log(chalk.bgWhite.black(':'), ...args)
    },
    info: (...args: any[]) => {
        console.log(chalk.bgWhite.black(':', ...args))
    },
    alert: (...args: any[]) => {
        console.log(chalk.bgYellow.black('!', ...args))
    },
    warn: (...args: any[]) => {
        console.log(chalk.bgYellow.black('!'), ...args)
    },
    error: (...args: any[]) => {
        console.log(chalk.bgRed.black('!'), ...args)
    }
}
```